samples/counter/README.md

```diff
@@ -33,9 +33,7 @@ Running Counter on iOS
 
 Run this:
 ```
-cd samples/counter/ios-uikit
-pod install
-open CounterApp.xcworkspace
+open samples/counter/ios-uikit/CounterApp.xcworkspace
```

Comment:
These are also .gitignore'd. Do we need to commit this now?

Reply:
If we switch it to `open samples/counter/ios-uikit/CounterApp.xcodeproj` it opens alright 🤷  Just gonna do that unless there's some benefit of `xcworkspace` here that I'm not seeing